packages/camera/camera/example/android/app/build.gradle

```diff
@@ -34,7 +34,7 @@ android {
     defaultConfig {
         applicationId "io.flutter.plugins.cameraexample"
         minSdkVersion 21
-        targetSdkVersion 28
+        targetSdkVersion 33
```

Comment:
Are the SDK version bumps necessary for this PR?

Reply:
Yes, the issue is only visible when you target SDK 33. If you leave 28 camera would work on Android 13.